LEAF_Request_Portal/templates/reports/LEAF_inbox_combined.tpl

```diff
@@ -1,287 +1,298 @@
+                    'top': 120
+                });
+            }
+        }, 100);
+        $('#headerTab').html('My Inbox');
+    });
 </script>
 <style>
-/*responsive grid*/
-.group:after,.section{clear:both}.section{padding:0;margin:0}.col{display:block;float:left;margin:1% 0 1% 1.6%}.col:first-child{margin-left:0}.group:after,.group:before{content:"";display:table}.group{zoom:1}.span_4_of_4{width:100%}.span_3_of_4{width:74.6%}.span_2_of_4{width:49.2%}.span_1_of_4{width:23.8%}@media only screen and (max-width:480px){.col{margin:1% 0}.span_1_of_4,.span_2_of_4,.span_3_of_4,.span_4_of_4{width:100%}}
+    /*responsive grid*/
+    .group:after, .section{clear:both}.section{padding:0;margin:0}.col{display:block;float:left;margin:1% 0 1% 1.6%}.col:first-child{margin-left:0}.group:after, .group:before{content:"";display:table}.group{zoom:1}.span_4_of_4{width:100%}.span_3_of_4{width:74.6%}.span_2_of_4{width:49.2%}.span_1_of_4
+
+    {width:23.8%}@media only screen and (max-width: 480px)
+
+    {.col{margin:1% 0}.span_1_of_4,.span_2_of_4,.span_3_of_4,.span_4_of_4{width:100%}}
 </style>
 <div id="genericDialog" style="visibility: hidden; display: none">
     <div>
         <div id="genericDialogbutton_cancelchange" style="display: none"></div>
         <div id="genericDialogbutton_save" style="display: none"></div>
-        <div id="genericDialogloadIndicator" style="visibility: hidden; z-index: 9000; position: absolute; text-align: center; font-size: 24px; font-weight: bold; background-color: #f2f5f7; padding: 16px; height: 400px; width: 526px"><img src="images/largespinner.gif" alt="loading..." /></div>
-        <div id="genericDialogxhr" style="width: 540px; height: 420px; padding: 8px; overflow: auto; font-size: 12px"></div>
+        <div id="genericDialogloadIndicator"
+             style="visibility: hidden; z-index: 9000; position: absolute; text-align: center; font-size: 24px; font-weight: bold; background-color: #f2f5f7; padding: 16px; height: 400px; width: 526px">
+            <img src="images/largespinner.gif" alt="loading..."/></div>
+        <div id="genericDialogxhr"
+             style="width: 540px; height: 420px; padding: 8px; overflow: auto; font-size: 12px"></div>
     </div>
 </div>
-
-<div id="loading" class="card" style="text-align: center; padding: 16px; font-size: 140%"><img src="images/largespinner.gif" alt="loading indicator" style="vertical-align: middle" /> Loading...</div>
+<div id="loading" class="card" style="text-align: center; padding: 16px; font-size: 140%"><img
+            src="images/largespinner.gif" alt="loading indicator" style="vertical-align: middle"/> Loading...
+</div>
 <div id="inboxContainer" style="display: none">
     <button id="btn_expandAll" class="buttonNorm" style="float: right">Expand all sections</button>
     <div class="section group">
-        <div class="col span_1_of_4">
-            <div id="index">Jump to section:
+        <div class="col span_1_of_4" style="margin-right: -5%">
```

Comment:
Another non-critical, but negative margins should be avoided whenever possible.  If is the last resort, it's fine, but better to find a way to fix overall CSS than add a negative margin.

Reply:
Yeah for this one specifically I just didn't want to break col span_1_of_4 for other scripts that could be using it. This is just a custom script for users to use and it needs to be customized specifically to that certain script so its unique in visual. If I edited the css for that class it would reflect in all other scripts. I guess I could of put in a new class for this one specifically, or even a new id.